apt.spec

```diff
@@ -4,7 +4,8 @@
 ## INITENV CMD_CSH if ( -f %{instroot}/common/apt-site-env.csh )  source %{instroot}/common/apt-site-env.csh; endif
 ## NOCOMPILER
 
-Source0: git://github.com/cms-sw/apt-rpm.git?obj=master/%{realversion}&export=apt-rpm-%{realversion}&output=/apt-rpm-%{realversion}.tgz
+Source0: https://github.com/cms-sw/apt-rpm/archive/%{realversion}.tar.gz 
+#git://github.com/cms-sw/apt-rpm.git?obj=master/%{realversion}&export=apt-rpm-%{realversion}&output=/apt-rpm-%{realversion}.tgz
```

Comment:
Had this worked at Fermilab I WOULD HAVE USED IT!!!!!
I think is only works from account with the cmsbuild credentials.